src/tests/plugins/VuePlugin.test.ts

```diff
@@ -0,0 +1,61 @@
+import { VuePlugin, RawPlugin } from "../../index";
+import { createEnv } from "../stubs/TestEnvironment";
+import { should } from "fuse-test-runner";
+
+const vueFileSource = `<template>
+    <div>
+        <p>{{ msg }}</p>
+        <input type="text" v-model="msg" />
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'app',
+        data () {
+            return {
+                msg: 'Welcome to Your Vue.js App'
+            }
+        }
+    }
+</script>
+`;
+
+export class VuePluginTest {
+    "Should return compiled vue code with render functions"() {
+        return createEnv({
+            project: {
+                files: {
+					"app.vue": vueFileSource
+                },
+                plugins: [
+                    [ VuePlugin(), RawPlugin({})]
+                ],
+                instructions: ">app.vue",
+            },
+        }).then((result) => {
+			const out = result.project.FuseBox.import("./app.vue").trim();
+			should(out).findString('_p.render');
+			should(out).findString('_p.staticRenderFns');
+			should(out).notFindString('_p.template');
+
+			should(out).equal(`var _p = {};
+var _v = function(exports){"use strict";
```

Comment:
🔬👍 good to add tests

this type of string output exactly matching unit test for something that may be a frequently updated dependency seems kind of fragile.

why not: 

- compile the content with fusebox (as it currently does) 
- write to disk & require, 
- or use the [node vm sandbox](https://nodejs.org/api/vm.html), 
- then [use vue unit testing](https://vuejs.org/v2/guide/unit-testing.html) to assert the rendered text? 

easier to read, update, and it asserts more things.